### DISCUSSION TITLE
replace voxelgi with hddagi instead sdfgi

### DISCUSSION THREAD

**Discussion by octanejohn** [godotengine/godot-proposals#10194] [3D]
make hddagi have 2 options to bake geometry in a box without cascades like voxelgi and open world one
while improving sdfgi(amd brixelizer?) as a mid-high end option
keeping the names as is for compatibility there basically correct

**Comment by AThousandShips** [comment]
See also:
* https://github.com/godotengine/godot-proposals/discussions/10180

**Reply by octanejohn** [reply]
ye i said amd brixelizer as in maybe an improvement part as to why it should not be replaced, this thread is more about not replacing sdfgi and instead replace voxelgi because is going be rewriten anyway

**Reply by Calinou** [reply]
SDFGI will be replaced either way as it has many limitations that can't be resolved (such as slow cascade generation speed).

A bounded GI implementation to supersede VoxelGI may be added in the future, but it's not guaranteed.
